src/layers/core/icon-layer/icon-layer.js

```diff
@@ -60,7 +60,8 @@ const defaultProps = {
   getPosition: x => x.position,
   getIcon: x => x.icon,
   getColor: x => x.color || DEFAULT_COLOR,
-  getSize: x => x.size || 1
+  getSize: x => x.size || 1,
+  getAngle: x => x.angle || 0
```

Comment:
I have no problem with either. Let's make it degrees for now as suggested by @Pessimistress